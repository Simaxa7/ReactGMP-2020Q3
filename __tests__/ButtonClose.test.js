import React from 'react';
import renderer from 'react-test-renderer';

import ButtonClose from '../src/components/button-close';

it('renders correctly 1', () => {
  const options = {
    classType: 'buttonCloseDefault',
    text: 'close',
    type: 'button',
  };

  const tree = renderer
    .create(<ButtonClose options={options} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly 2', () => {
  const options = {
    classType: 'buttonActionDefault',
    text: 'action',
  };

  const tree = renderer
    .create(<ButtonClose options={options} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
