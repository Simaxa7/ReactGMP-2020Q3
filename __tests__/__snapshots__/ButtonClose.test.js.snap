// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1 1`] = `
<button
  className="button-close-default button-close"
  type="button"
>
  close
</button>
`;

exports[`renders correctly 2 1`] = `
<button
  className="button-action-default button-action"
  type="button"
>
  action
</button>
`;
